hphp/test/slow/include/fast-stat-cache.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$file = tempnam('/tmp', 'test-fast-stat-cache');
+file_put_contents($file, "<?php\necho 'Old';");
+include($file);
+unlink($file);
+sleep(1);
```

Comment:
I add a new commit that reduce a sleep() call.

_If we clear the cache any time we make changes to the file system (create/delete a file, as opposed to checking existence), would things work better in this aspect?_

I had considered this idea, but there are so many ways to change a file. File system functions like fwrite, other extension function like Imagick::writeImage, and even another process may change the file. Inotify can handle all this case(that's what StatCache do), but it's slow, after turning on Server.StatCache the performance decrease by 5% on wordpress.